src/main/app/paths.ts

```diff
@@ -2,7 +2,7 @@ import { RoutablePath } from 'common/router/routablePath'
 
 export class Paths {
   static readonly homePage = new RoutablePath('/', false)
-  static readonly otherOptionsPage = new RoutablePath('/other-options', false)
```

Comment:
not for this PR, but maybe it would be a good idea to unify naming convention. Sometimes in this class we have `blablaPage` and sometimes just `blabla`. 

Either or.